eventkit_cloud/tasks/task_factory.py

```diff
@@ -188,12 +200,47 @@ def create_task(export_provider_task_uid=None, stage_dir=None, worker=None, sele
     :param export_provider_task_uid: An export provider task UUID.
     :param worker: The name of the celery worker assigned the task.
     :return: A celery task signature.
-    """""
-    task_map = {"bounds": bounds_export_task, "selection": output_selection_geojson_task, "zip": zip_export_provider}
```

Comment:
This removes support for the zip_export_provider.  This is a feature which allows an administrator to select the zip file option in the admin console to zip up the files associated with the ExportProvider.